astronomer/providers/great_expectations/operators/great_expectations.py

```diff
@@ -203,16 +202,17 @@ def build_runtime_datasources(self) -> Dict[str, Any]:
                 },
             }
         else:
-            datasource_config["default_datasource"]["execution_engine"] = {
-                "class_name": self.execution_engine
-            }
-            datasource_config["default_datasource"]["data_connectors"] = {
-                "default_configured_data_connector_name": {
-                    "class_name": "ConfiguredAssetFilesystemDataConnector",
-                    "base_directory": self.data_context_root_dir,
-                    "default_regex": self.file_regex,
-                },
-            }
+            pass
```

Comment:
Maybe this should throw and error - something about not recognizing the data source and needing further configuration?